Packs/1Password/Integrations/OnePasswordEventCollector/OnePasswordEventCollector.yml

```diff
@@ -0,0 +1,117 @@
+category: Analytics & SIEM
+commonfields:
+  id: OnePasswordEventCollector
+  version: -1
+configuration:
+- defaultvalue: https://events.1password.com
+  display: Server URL
+  additionalinfo: The API server URL depends on the domain where the account is hosted. Refer to the integration Help section for more details.
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- displaypassword: API Token
+  additionalinfo: The bearer token used to authenticate with the 1Password Events API. This must include the required features (scopes) that correspond to the event types to be fetched.
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+  section: Connect
+- display: Trust any certificate (not secure)
+  additionalinfo: Allow connections without verifying the SSL certificate of the server.
+  name: insecure
+  type: 8
+  required: false
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+  section: Connect
+  advanced: true
+- display: Fetch Events
+  additionalinfo: Whether to fetch events from 1Password.
+  name: is_fetch_events
+  type: 8
+  required: false
+  section: Collect
+- display: Types of events to fetch
+  name: event_types
+  type: 16
+  options:
+  - Audit events
+  - Item usage actions
+  - Sign in attempts
+  required: true
+  section: Collect
+- defaultvalue: 2 weeks ago
+  display: First fetch time
+  additionalinfo: 'First event timestamp to fetch. For example: "1 min ago", "2 weeks ago", "3 months ago".'
+  name: first_fetch
+  type: 0
+  required: false
+  section: Collect
+- defaultvalue: 1000
+  display: Maximum number of audit events per fetch
+  name: audit_events_limit
+  type: 0
+  required: true
+  section: Collect
+  advanced: true
+- defaultvalue: 1000
+  display: Maximum number of item usage actions per fetch
+  name: item_usage_actions_limit
+  type: 0
+  required: true
+  section: Collect
+  advanced: true
+- defaultvalue: 1000
+  display: Maximum number of sign in attempts per fetch
+  name: sign_in_attempts_limit
+  required: true
+  type: 0
+  section: Collect
+  advanced: true
+description: 'Fetch events about actions performed by 1Password users within a specific account, access and modifications to items in shared vaults, and user sign-in attempts.'
+display: 1Password Event Collector
+name: OnePasswordEventCollector
+script:
+  commands:
+  - arguments:
+    - description: 'The maximum number of events to fetch for the given event type.'
+      name: limit
+      required: false
+      defaultValue: 1000
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: Set this argument to True in order to push events to XSIAM, otherwise the command will only display them.
```

Comment:
```suggestion
      description: Set this argument to True in order to push events to Cortex XSIAM, otherwise the command will only display them.
```

Reply:
Fixed